apps/root/src/services/conctactListService.ts

```diff
@@ -1,11 +1,18 @@
-import { Contact, ContactList, IContactListService } from '@types';
+import { AccountLabelsAndContactList, Contact, ContactList, IContactListService } from '@types';
 import ProviderService from './providerService';
 import ContractService from './contractService';
 import MeanApiService from './meanApiService';
-import { findIndex, remove } from 'lodash';
+import { map } from 'lodash';
 import AccountService from './accountService';
+import WalletService from './walletService';
+import LabelService from './labelService';
+import { EventsManager } from './eventsManager';
 
-export default class ContactListService implements IContactListService {
+export interface ContactListServiceData {
```

Comment:
I'm not happy about this, but I cant figure out a better way to do this

Reply:
Having a single prop in storage, as we do now, this is not entirely necessary, but I liked the idea of having this set up already. Let me know if you have anything in mind!